SQL/CRUD/CRUDHomework3.sql

```diff
@@ -0,0 +1,8 @@
+SELECT `customer`.*
+FROM `customer`
+JOIN `agents`
+ON `customer`.`AGENT_CODE` = `agents`.`AGENT_CODE`
+WHERE `customer`.`GRADE` = 3
+AND NOT `customer`.`CUST_COUNTRY` = "India"
+AND `customer`.`OPENING_AMT` < 7000
+AND `agents`.`COMMISSION` < 0.12
```

Comment:
You should add the semicolon at the end.

_Fact_: You are working on the GUI (Graphic User Interface), but in the development environment, you will face debugging on the server which has no UI to interact with. You have to run your query on the CLI (Command Line Interface). Then if have no semicolon at the end of a query, it's make you some trouble.
As you can see, I can't run the query unless the semicolon

![image](https://user-images.githubusercontent.com/83938068/118764197-52026200-b8a3-11eb-8ec2-9696c4d98d48.png)